simulation-scripts/perturb.sh

```diff
@@ -513,12 +515,12 @@ run_file_on_host() {
   #shellcheck disable=SC2094
   (
     touch "${TMP_DIR}/perturb-script-file-${HOST}.log"
+    get_file_to_run "${FILE}" >> "${TMP_DIR}/perturb-script-file-${HOST}.log"
     exec 19>>"${TMP_DIR}/perturb-script-file-${HOST}.log"
```

Comment:
Why is this a magic number 19?  what happens if there are 19 open fds, why not use a much higher number?  Either way a descriptive variable name would be nicer :) /cc @sublimino

Reply:
Using a named file descriptor now instead and actually closing it once I'm done